src/index.html

```diff
@@ -12,13 +12,13 @@
 <body>
     <h1>Leaderboard</h1>
     <main class="content">
-    <article>
+    <article class="board">
    <div class="align">
         <h2 class="table-title">Recent scores</h2>
         <button type="button" class="refresh-button">Refresh</button>
```

Comment:
👏🎉 Good job so far! Your project is coming along nicely. 🌟 However, there's a small improvement you can make to enhance the user experience.

- [ ] [Optional]💡🔄 Currently, after clicking the submit button, the user needs to manually refresh the page to update the score list. But you can make it more user-friendly! 😊 After successfully adding the value to the API, you can call your `this.fetchScoresAndShowRecords()` function to automatically update the score list and display the new entry. This way, the user won't have to go through the extra step of refreshing the page.


🚀💻 By implementing this enhancement, you'll make your project even more seamless and user-friendly. Keep up the great work, and good luck with the rest of your project! You're doing fantastic! 💪🎉

Reply:
Thanks for the review, even thou it's optional but the project requirements needs data to be summoned from the API only when the refresh button is clicked.